.circleci/config.yml

```diff
@@ -15,12 +15,18 @@ executors:
   integration-test-environment:
     docker:
       - image: cimg/openjdk:21.0
-# For future use - change ITs to use a real postgres instance rather than H2
```

Comment:
We always thought we'd end up making this change sooner or later! 😁 
The approach we've used has been copied from Send Legal Mail